environment/environment.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+
+import numpy as np
+import matplotlib.pyplot as plt
+import pdb
+import random
+
+'''Initialize agent environment'''
+world = np.full((100,100), float(0))
+
+'''Initialize agent motion'''
+def agent_up(i,j):
+	for x in range(len(world)):
+		#pdb.set_trace()
+		if j >= 100:
+			j = 100
+		elif j <= 0:
+			j = 0
+		else:
+			j = j + 1
+		plt.plot(i,j,'ro')
+		plt.show()
+
+def agent_down(i,j):
+	for x in range(len(world)):
+		if j >= 100:
+			j = 100
+		elif j <= 0:
+			j = 0
+		else:
+			j = j - 1
+			plt.plot(i,j,'ro')
+			plt.show()
+
+def test_down():
+	assert agent_down(5,-1) == (5,0)			
```

Comment:
So right now you are only performing one test on your function. I think it is suggested to perform multiple tests. Along with that, giving a brief definition of what they test is doing/the output is really helpful.